### ISSUE TITLE
PTLC Cycle Jamming

### ISSUE THREAD

**Issue by LLFourn** [bitcoin-research/bitcoin-research.github.io#1]
*Discussion Issue for the [PTLC Cycle Jamming](https://bitcoin-research.github.io/problems/ptlc-cycle-jamming.html) problem*

**Comment by gijswijs** [comment]
> This attack is easily preventable with HTLC based payments where the lock is the same SHA256 image at each hop.

That is true. However, I am unaware of any client currently preventing payment loops. So the current state of affairs is that Cycle Jamming is already possible. Cristina Pérez-Solà wrote a paper [1] describing the attack. Maybe we should add this to the Related Research?

[1] Pérez-Solà, C., Ranchal-Pedrosa, A., Herrera-Joancomartí, J., Navarro-Arribas, G., & Garcia-Alfaro, J. (2020). LockDown: Balance Availability Attack Against Lightning Network Channels. https://doi.org/10.1007/978-3-030-51280-4_14

**Comment by gijswijs** [comment]
Preprint available here: https://eprint.iacr.org/2019/1149

**Comment by LLFourn** [comment]
Thanks @gijswijs -- this is an extremely valuable contribution to this research topic. They basically show that these cycling attacks to in practice enhance jamming attacks significantly.

See PR: https://github.com/bitcoin-problems/bitcoin-problems.github.io/pull/18
